### PR TITLE
[Waste] Add cheque reference field.

### PR DESCRIPTION
**Pull request by dracos**
[skip changelog]

### CODE REVIEW

File: templates/web/base/admin/reports/_edit_waste.html [hunk 1]
```diff
@@ -161,6 +161,7 @@ <h2>Payment information</h2>
     [ 'Pro rata payment', 'pro_rata' ],
     [ 'Admin fee', 'admin_fee' ],
     [ 'Payment method', 'payment_method' ],
+    [ 'Cheque reference', 'chequeReference', 1 ],
     [ 'DD reference', 'payerReference', 1 ],
     [ 'SCP reference', 'scpReference', 1 ],
     [ 'Last payment method', 'LastPayMethod' ],
```
Comment (struan) [inline templates/web/base/admin/reports/_edit_waste.html:167]: Unrelated to this but I notice that this just lists the number in the admin which I'm not sure is entirely helpful?

**Review by struan — CHANGES_REQUESTED** [review]
One small bug but otherwise this looks good.

Only other things is it might be an idea to list the cheque reference on the summary page?

File: web/js/waste.js [hunk 2]
```diff
@@ -6,6 +6,9 @@ $(function() {
         $btn.parents('.govuk-form-group').addClass('loading');
     });
 
+    var id = document.getElementById('form-cheque_reference-row');
+    if (id) { id.className += ' hidden-js'; }
```
[the remark this answers is not in the document]
Comment (dracos) [reply web/js/waste.js:10]: Ah, it needs to run before the actual data-show/hide set up code (which does check which item is selected), hmm.

**Review by struan — APPROVED** [review]
👍

**Pull request merged**